static/app/components/events/interfaces/spans/anchorLinkManager.tsx

```diff
@@ -1,56 +0,0 @@
-import {Component, createContext} from 'react';
```

Comment:
curious why this file's removed. is it not needed anymore?

Reply:
It's been renamed to `spanContext`, which handles the state management for all spans currently expanded in addition to the stuff that `anchorLinkManager` was doing